src/main/java/com/google/api/codegen/configgen/DiscoConfigGeneratorApp.java

```diff
@@ -72,22 +74,19 @@ protected void process() throws Exception {
   }
 
   /** Initializes the Discovery document document. */
-  private Document setupDocument() {
+  public static Document setupDocument(String discoveryDocPath, DiagCollector diagCollector) {
```

Comment:
Why does this need to be public?

Reply:
oh yes this can be reverted back to `private static`. thanks